_pages/algorithm-development/source-control-management.md

```diff
@@ -12,6 +12,7 @@ redirect_from:
   - /algorithm-development/algorithm-basics/git/
   - /algorithm-development/git-support/
   - /algorithm-development/git/
+  - /algorithmia-enterprise/scms
```

Comment:
@warmbowski redirect added here; I checked and it works locally

Reply:
ah, that's what that is.